scripts/deploy/oauth-smoke.sh

```
@@ -0,0 +1,239 @@
+                process.exit(1);
+              }
+              const auth = parsed?.auth ?? {};
+              const clientId = typeof auth.clientId === "string" ? auth.clientId : "";
+              const redirectUri =
+                typeof auth.redirectUri === "string" ? auth.redirectUri : "";
+              if (!clientId || !redirectUri) {
+                console.log("missing-auth-config-fields");
+                process.exit(1);
+              }
+              process.stdout.write(`${clientId}\n${redirectUri}`);
+            '
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)